hivemind/p2p/p2p_daemon.py

```diff
@@ -4,15 +4,17 @@
 from collections.abc import AsyncIterable as AsyncIterableABC
 from contextlib import closing, suppress
 from dataclasses import dataclass
+from google.protobuf.message import Message
```

Comment:
This import should be moved to the next group (third-party imports). Otherwise, `isort` won't accept it.